includes/Admin/Tools.php

```diff
@@ -354,6 +354,11 @@ class="button-secondary"/>
 							</tr>
 							</tbody>
 						</table>
+
+            <h4><span><?php esc_html_e( 'Additional Options', 'cp-library' ); ?></span></h4>
+            <input type='checkbox' checked id='sideload-audio-urls' name='sideload-audio-urls'>
+            <label for='sideload-audio-urls'>Sideload Audio URLs</label>
+
```

Comment:
Update label to say "Attempt to import mp3 files to the Media Library", and be sure to add i18n.